modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,139 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3_{context}"]
+= Release notes for {gitops-title} 1.3
+
+{gitops-title} 1.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="support-matrix-1-3_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
+[cols="1,1",options="header"]
+|===
+| Feature | {gitops-title} 1.3.0
+| Argo CD
+| GA
+| Argo CD ApplicationSet
+| TP
+| {gitops-title} Application Manager (kam)
+| TP
+| {gitops-title} Service
+| TP
+|===
+
+[id="dependencies-versions-1-3_{context}"]
+== Dependencies Versions
+
+Please note the upgraded versions for the following dependencies of GitOps v1.3.0:
+
+.Dependencies Version
+[cols="1,1",options="header"]
+|===
+| Dependency | Version
+| ApplicationSet
+| v0.2.0
+| Kustomize
+| v4.2.0
+| Argo CD
+| v2.1.2
+| Helm
+| v3.6.0
+|===
+
+[id="new-features-1-3_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.0:
+
+* After Argo CD is installed, Red Hat Single Sign On (SSO) is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Red Hat SSO installation after the operator is installed which will result in removing the Red Hat SSO deployment from the `openshift-gitops` namespace.
+Additionally, for any new Argo CD instance you can enable Red Hat SSO by setting the flag to true; when the flag is not present, Red Hat SSO will be disabled by default.
+
+* The default Argo CD instance installed by the OpenShift GitOps operator as well as accompanying controllers can now be made to run on the cluster's infrastructure nodes by setting a simple configuration toggle.
+
+[discrete]
+.Procedure
```

Comment:
When I view in rich text, this doesn't feel indented under the second bullet point, could you please verify?

Reply:
@DebarghoGhosh adding a '+' above the [discrete] will solve the indentation issue. 
But more importantly, we should not have procedures in new features, these should be just key highlights in bulleted points. I would suggest moving this procedure to an appropriate assembly in the GitOps section.
However, please note, procedures as an exception can be seen in the Known Issues section.

Reply:
@Preeticp as per discussion, removing the procedure from the New Features section and we can add it in the following page:

https://deploy-preview-37968--osdocs.netlify.app/openshift-enterprise/latest/cicd/gitops/configuring_argo_cd_to_recursively_sync_a_git_repository_with_your_application/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.html

@reginapizza  please check and confirm if we can add the procedure in the above page

Reply:
that's fine with me, I agree with that decision if procedures shouldn't be in new features sections